src/server/components/lists/find/helpers/getRedirectIfListIsEmpty.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  formatCountryParam,
+  getCountryFuneralDirectorsRedirectLink,
+  getCountryLawyerRedirectLink,
+  getCountryTranslatorsInterpretersRedirectLink,
+} from "server/components/lists/helpers";
+import { some } from "server/models/listItem/providers/helpers";
```

Comment:
I appreciate you probably didn't write this function but I'm not sure the name `some` is super descriptive.

Do you think it's worth updating the function name?

Reply:
It would probably make more sense if this was a class / module rather than direct import. e.g. `listItem.providers.some`?

Reply:
Yes, `providers.some` make more sense. 

`listItem.providers.some` would make even more sense but it's a step in the right direction 👍